overrides/scripts/ExtendedCrafting.zs

```diff
@@ -6,6 +6,9 @@ import scripts.CommonVars.makeExtremeRecipe5 as makeExtremeRecipe5;
 import scripts.CommonVars.makeExtremeRecipe7 as makeExtremeRecipe7;
 import scripts.CommonVars.makeExtremeRecipe9 as makeExtremeRecipe9;
 
+<extendedcrafting:frame>.displayName = "Black Steel Frame";
+<extendedcrafting:lamp>.displayName = "Black Steel Lamp";

```

Comment:
The Glowstone black iron lamp is meta ID 0, but there is also a Luminessence variant with meta ID 1 that still has the old name.